src/components/PurchaseOrder/UpdateOrderErrorModal/UpdateOrderErrorModal.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import PropTypes from 'prop-types';
+
+import { Button, Modal } from '@folio/stripes/components';
+
+const UpdateOrderErrorModal = ({ orderNumber, cancel, reason }) => {
```

Comment:
`reason` is not actually a reason, it's a reasonKey or something

Reply:
renamed with errorCode